### PR TITLE
Updated reference page

### PR DESCRIPTION
**Pull request by Shruti025**
Issue: #1168 

-Fixed the bug
-Made the required name and image changes in the cards.

### CODE REVIEW

**Review by anushbhatia — COMMENTED** [review]
image
Mobile view the images are getting squished (the ratio of image is changed)

File: assets/js/references.js [hunk 1]
```diff
@@ -128,7 +128,7 @@ const references = [
     image2: "assets/images/references-page/reviewphoto4.png",
     message2:
       "I had worked with John in research intern team, Flipkart. John would be an asset to have on any team. To say he's a team leader would be an understatement. during his tenure at our company, he's gone out of his way to welcome newcomers and resolve any problems they might have. He pitches innovative ideas at company meetings but also listens to what others have to say, including constructive criticism. When budgetary constraints limited the work our company could complete, John put his highly creative mind to use to come up with alternatives.",
```
Comment (smaranjitghose) [inline assets/js/references.js:130]: Can you please remove the unused code

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by Shruti025** [comment]
I'll work on the reviews today and submit a PR by tomorrow

**Comment by anushbhatia** [comment]
Will create another issue for responsiveness

**Pull request merged**